qiskit/pulse/instructions/instruction.py

```diff
@@ -190,7 +190,9 @@ def shift(self: ScheduleComponent, time: int, name: Optional[str] = None) -> Sch
         """
         if name is None:
             name = self.name
-        return Schedule((time, self), name=name)
```

Comment:
Since `Instruction` is also a subclass of `ScheduleComponent`, I went ahead and removed the usage of the deprecated argument.
I can obviously roll-back or handle this according to your guidelines.